### ISSUE TITLE
Conflict with third party

### ISSUE THREAD

**Issue by EasonLex** [OneSignal/react-native-onesignal#349]
Hi, I met a problem that OneSignal has conflict issue with third parties what I used.
One is Bugsnag, the other one is RNShare.

Did anyone had met this issue and got some solutions?

This issue only appeared on Android, the iOS part is work perfectly.

**Comment by jkasten2** [comment]
@EasonLex Can you provide more details? Are you seeing specific build or runtime errors you can attach here? Can you provide steps to reproduce the issue?

**Comment by EasonLex** [comment]
@jkasten2 Thanks for your reply.

There is no build error but runtime error that when I launch the app, app will crash immediately. 
I removed Bugsnag and RNShare component, it worked (no more crash) and I do success with receiving the push notification from OneSignal.

https://docs.bugsnag.com/platforms/react-native/
https://www.npmjs.com/package/react-native-share

I thought maybe it's the same issue with react-native-map, and I tried to use the way similar with that, but the app still crash.
